pkg/csi/service/wcpguest/controller_helper.go

```diff
@@ -65,19 +65,32 @@ func validateGuestClusterCreateVolumeRequest(ctx context.Context, req *csi.Creat
 	}
 	// Get create params
 	var supervisorStorageClass string
+	var storageTopologyTypeValue string
 	params := req.GetParameters()
-	for param := range params {
-		paramName := strings.ToLower(param)
-		if paramName != common.AttributeSupervisorStorageClass {
-			msg := fmt.Sprintf("Volume parameter %s is not a valid GC CSI parameter", param)
+	if !commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.TKGsHA) {
+		for param := range params {
+			paramName := strings.ToLower(param)
+			if !(paramName == common.AttributeSupervisorStorageClass ||
```

Comment:
it would be easier to understand the logic if reformat the condition check, 

```
paramName != common.AttributeSupervisorStorageClass && paramName != common.AttributeStorageTopologyType
```